### PR TITLE
Some enhancements and some bugfixes - bumpt to a new Version

### LINKED ISSUE #21: Bump Version to the lates commit
The last Release (also used for haikuporter) covers only the commits up to 2014 and not the latest changes (especially the packagemanagement adjustments to the path)

### PR DESCRIPTION
**Pull request by Paradoxianer**
This should fix #21 #20 #18 
And also improves the search window (now shows a paragraph of the found bible text - not only the verse).
You can select multiple search entries and copy them.

### CODE REVIEW

File: ScriptureGuide/Makefile [hunk 1]
```diff
@@ -90,26 +96,26 @@ OPTIMIZE := FULL
...
-LOCALES = 
+LOCALES = en de
 
 #	Specify all the preprocessor symbols to be defined. The symbols will not
 #	have their values set automatically; you must supply the value (if any) to
 #	use. For example, setting DEFINES to "DEBUG=1" will cause the compiler
 #	option "-DDEBUG=1" to be used. Setting DEFINES to "DEBUG" would pass
 #	"-DDEBUG" on the compiler's command line.
-DEFINES =
+DEFINES = DEBUG=1
```
Comment (waddlesplash) [inline ScriptureGuide/Makefile:117]: Please remove these changes (i.e. enabling DEBUG and SYMBOLS.)

Comment (waddlesplash) [reply ScriptureGuide/Makefile:117]: (and also DEBUGGER.)

File: ScriptureGuide/LogosSearchWindow.cpp [hunk 2]
```diff
@@ -342,6 +354,35 @@ void SGSearchWindow::MessageReceived(BMessage *message)
...
+			if (be_clipboard->Lock()) {
+				be_clipboard->Clear();
+    			if (clip = be_clipboard->Data()) {
+    				clip->AddData("text/plain", B_MIME_TYPE, clipBoardString.String(),clipBoardString.Length());
+    				be_clipboard->Commit();
+    			}
+    			else
+    				printf("ERROR couldnt get clipboard Data");
+    			be_clipboard->Unlock();
+ 			}
```
Comment (waddlesplash) [inline ScriptureGuide/LogosSearchWindow.cpp]: Please clean up the coding style in this section.

File: ScriptureGuide/SwordBackend.cpp [hunk 3]
```diff
@@ -198,13 +233,18 @@ vector<const char*> SGModule::SearchModule(int searchType, int flags, const char
...
 	vector<const char*> results;
 	
+	
 	int chapter = ChaptersInBook(endbook);
 	int verse = VersesInChapter(endbook,chapter);
 	
 	BString searchstr;
 	searchstr << startbook << " 1:1-" << endbook << " " << chapter << ":" << verse;
-	
+	printf("Searching %s in %s\n",searchText,searchstr.String());
```
Comment (waddlesplash) [inline ScriptureGuide/SwordBackend.cpp]: Remove debug print

File: ScriptureGuideManager/MainWindow.cpp [hunk 4]
```diff
@@ -32,21 +33,39 @@ MainWindow::MainWindow(BRect frame)
...
+			
+				
+			row->SetField(install_field,0);
+			row->SetField(book_field,1);
+			row->SetField(lang_field,2);
+			fBookListView->AddRow(row);
+			
+			//	row->SetMarked(true);
+			/*BookRow *BookRow=new BookRow(cfile->fDescription.String(),cfile);
+			fListView->AddItem(BookRow);*/
```
Comment (waddlesplash) [inline ScriptureGuideManager/MainWindow.cpp:67]: Remote commented code

File: ScriptureGuideManager/Makefile [hunk 5]
```diff
@@ -93,19 +93,19 @@ LOCALES =
 #	use. For example, setting DEFINES to "DEBUG=1" will cause the compiler
 #	option "-DDEBUG=1" to be used. Setting DEFINES to "DEBUG" would pass
 #	"-DDEBUG" on the compiler's command line.
-DEFINES = 
+DEFINES = DEBUG=1
```
Comment (waddlesplash) [inline ScriptureGuideManager/Makefile]: Same comments about debug here

**Review by waddlesplash — CHANGES_REQUESTED** [review]

**Comment by Paradoxianer** [comment]
thanks for all the hints sorry that i forgot the debug stuff :( ... will fix them soon...  i think i will fill  a complete new Pull Request since i found time to add some more features (like sorted SGManager) and localisation.

**Review by Paradoxianer — COMMENTED** [review]
This should fixes the Codesytel violations, makefile debug setting and add a lot more features...

**Pull request closed without merging**